common/scripted_modifiers/wc_scheme_scripted_modifiers.txt

```diff
@@ -0,0 +1,54 @@
+﻿scheme_base_success_chance_modifier = { #Use MULTIPLIER if you want harsher/softer penalties
+		# Spiritual Head of Faith
+		modifier = {
+			add = -35
+			desc = "SCHEME_VS_SPIRITUAL_HOF"
+			scope:target = {
+				faith = scope:owner.faith
+				faith = {
+					has_doctrine = doctrine_spiritual_head
+				}
+				faith = {
+					exists = religious_head
+					religious_head = {
+			 			this = scope:target
+			 		}
+				}
+			}
+		}
+		
+		# At War
```

Comment:
Also, I would split these 2: `Spiritual Head of Faith` and `At War`. They differ in murder and abdication, so by uniting them you make these scripted modifiers unusable in these 2.